aiida_firecrest/transport.py

```diff
@@ -315,45 +564,75 @@ def copy(
         dereference: bool = False,
         recursive: bool = True,
     ) -> None:
+        """Copy a file or directory on the remote. FirecREST does not support symlink copying.
+
+        :param recursive: If True, copy directories recursively.
+        note that the non-recursive option is not implemented in FirecREST server.
+        And it's not used in upstream, anyways...
+
+        :param dereference: If True, copy the target of the symlink instead of the symlink itself.
+        """
+        # TODO: investigate overwrite (?)
+
         if not recursive:
             # TODO this appears to not actually be used upstream, so just remove there
             raise NotImplementedError("Non-recursive copy not implemented")
-        source = self._cwd.joinpath(remotesource).enable_cache()
-        destination = self._cwd.joinpath(remotedestination).enable_cache()
+        if dereference:
+            raise NotImplementedError(
+                "Dereferencing not implemented in FirecREST server"
+            )
+        source = self._cwd.joinpath(
+            remotesource
+        )  # .enable_cache() it's removed from from path.py to be investigated
+        destination = self._cwd.joinpath(
+            remotedestination
+        )  # .enable_cache() it's removed from from path.py to be investigated
```

Comment:
unclear, please consider to make an issue

Reply:
Alright, just opened an issue is opened on this:
https://github.com/aiidateam/aiida-firecrest/issues/41